X-CASH-official/desktop-wallet
# Fix mac os build

# Description

The PR contains: 
- fixes of Mac OS build and wallet creation;
- corrections of JS/TS code style for wallet-rpc service;
- introduces `eslint` for the project;
- adds more precise logging of `xcash-wallet-rpc` and HTTP request / responses between the electron app and `xcash-wallet-rpc`;
- updates dependencies;
 
In the future PRs it's better to apply formatting for the files where changes are made.
## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. Run `npm run package-mac`;
2. Create a new wallet and see something similar to this:

<img width="1552" alt="Screenshot 2021-02-16 at 22 18 20" src="https://user-images.githubusercontent.com/7963446/108133777-c4ee1800-70bd-11eb-9e84-0700366c950f.png">

<img width="1552" alt="Screenshot 2021-02-17 at 1 39 26" src="https://user-images.githubusercontent.com/7963446/108135565-ffa57f80-70c0-11eb-998f-c0a988de760b.png">


**Test Configuration**:
* Firmware version: Mac OS BigSur (version 11.2 (20D64))
* Hardware: MacBook Pro (13-inch, 2019, Four Thunderbolt 3 ports)
  

# Checklist:


~- [ ] I have added tests that prove my fix is effective or that my feature works~
~- [ ] New and existing unit tests pass locally with my changes~
